### DISCUSSION TITLE
[wn-translate-plugin] Automatic import from CSV in theme folder

### DISCUSSION THREAD

**Discussion by matteotrubini** [wintercms/winter#124] [Ideas]
Manage tranlsation trough backend is simple and useful, but sometime is less immediate than an (S)FTP/RSYNC upload.

Proposal:
- create console command to permit execution of import / export programmatically - it's just a link to existing routines
- add a routine to automatically load/update translation:
    - add a boolean option to enable autoloading - this is to preserve current behavior, no impact on performance
    - check if a csv in theme folder exists
    - check if it is newer than the last one loaded (timestamp saved somewhere in the db)
    - load/update it using the already available import routine
    - delete it upon success
    - maybe clear cache?

**Comment by mjauvin** [comment]
Sure, are you willing to submit a PR to do this?

**Reply by matteotrubini** [reply]
Too busy at the moment. But I can help in review / testing and participate in a reward.
